src/Libraries/Nop.Services/ExportImport/ImportManager.cs

```diff
@@ -2183,6 +2387,340 @@ public virtual async Task ImportCategoriesFromXlsxAsync(Stream stream)
+                    {
+                        await _addressService.InsertAddressAsync(orderAddress);
+
+                        if (order.PickupInStore)
+                            order.PickupAddressId = orderAddress.Id;
+                        else
+                            order.ShippingAddressId = orderAddress.Id;
+                    }
+                    else
+                        await _addressService.UpdateAddressAsync(orderAddress);
+                }
+                else
+                    order.ShippingAddressId = null;
+
+                //set some default values if not specified
+                if (isNew)
+                {
+                    //affiliate
+                    var affiliate = await _affiliateService.GetAffiliateByIdAsync(customer.AffiliateId);
+                    if (affiliate != null && affiliate.Active && !affiliate.Deleted)
+                        order.AffiliateId = affiliate.Id;
+
+                    //checkout attributes
+                    order.CheckoutAttributesXml = await _genericAttributeService.GetAttributeAsync<string>(customer, NopCustomerDefaults.CheckoutAttributes, storeId);
+                    order.CheckoutAttributeDescription = await _checkoutAttributeFormatter.FormatAttributesAsync(order.CheckoutAttributesXml, customer);
+
+                    //customer language
+                    var customerLanguage = await _languageService.GetLanguageByIdAsync(customer.LanguageId ?? 0);
+                    if (customerLanguage == null || !customerLanguage.Published)
+                        customerLanguage = await _workContext.GetWorkingLanguageAsync();
+                    order.CustomerLanguageId = customerLanguage.Id;
+
+                    //tax display type
+                    if (_taxSettings.AllowCustomersToSelectTaxDisplayType)
+                        order.CustomerTaxDisplayType = customer.TaxDisplayType ?? TaxDisplayType.IncludingTax;
+                    else
+                        order.CustomerTaxDisplayType = _taxSettings.TaxDisplayType;
+
+                    //set other default values
+                    order.AllowStoringCreditCardNumber = false;
+                    order.AuthorizationTransactionCode = string.Empty;
+                    order.AuthorizationTransactionId = string.Empty;
+                    order.AuthorizationTransactionResult = string.Empty;
+                    order.CaptureTransactionId = string.Empty;
+                    order.CaptureTransactionResult = string.Empty;
+                    order.CardCvv2 = string.Empty;
+                    order.CardExpirationMonth = string.Empty;
+                    order.CardExpirationYear = string.Empty;
+                    order.CardName = string.Empty;
+                    order.CardNumber = string.Empty;
+                    order.CardType = string.Empty;
+                    order.CustomerIp = string.Empty;
+                    order.CustomerTaxDisplayTypeId = (int)order.CustomerTaxDisplayType;

```

Comment:
this assignment is redundant, it duplicates the line order.CustomerTaxDisplayType = _taxSettings.TaxDisplayType;